src/tests/bundles/bionic-queens-swift-v2.yaml

```diff
@@ -33,7 +33,11 @@ relations:
 - - cinder-ceph:storage-backend
   - cinder:storage-backend
 - - cinder
+<<<<<<< HEAD
+  - cinder-backup-swift-proxy
+=======
   - cinder-backup-swift
+>>>>>>> master
```

Comment:
There appears to be one more conflict to resolve

Reply:
Fixed